### PR TITLE
Fix versions in requirements.txt for python 3.5 and 3.9

### PR DESCRIPTION
**Pull request by jsoriano**
## What does this PR do?

Fix versions of packages in `requirements.txt` file so they can be installed in all python versions from 3.5 to 3.9.

Added a cleaned-up version of a one-liner I was using to check the `requirements.txt` file with different versions.

## Why is it important?

Packages listed in current `requirements.txt` file cannot be installed in python 3.5 and 3.9:
* Pillow package doesn't have a candidate for 3.9, it was removed in #20407, but added again by mistake in #16883.
* `zipp` package needed by `pytest` works with different versions depending the version of python, version that works with python 3.5 doesn't work with other versions.

We still need to support python 3.5 because it is used in some CI workers.

### CODE REVIEW

File: script/check_python_requirements.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+VERSIONS=${VERSIONS:-3.5 3.6 3.7 3.8 3.9-rc}
+REQUIREMENTS=${1:-${BEATS_PATH}/libbeat/tests/system/requirements.txt}
+
+if [ ! -f $REQUIREMENTS ]; then
+	echo "Requirements file doesn't exist: $REQUIREMENTS"
+	exit -1
+fi
+
+REQUIREMENTS=$(realpath $REQUIREMENTS)
```
Comment (ChrsMark) [inline script/check_python_requirements.sh:25]: Will this script work out of-the-box on other environments except linux? On darwin it seems that there is no such command by default (but can be installed with `brew install coreutils`): https://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-os-x

Comment (jsoriano) [reply script/check_python_requirements.sh]: Good point, I have added a small function that uses `cd` and `pwd` as suggested in one of the comments there.

**Comment by jsoriano** [comment]
Failing test is fixed in master, merging.

**Comment by jsoriano** [comment]
Will apply the changes in 7.x branch on the backport PR for pytest https://github.com/elastic/beats/pull/20478

**Pull request merged**